### ISSUE TITLE
Implement to/from_raw fns for heavy data structures such as ProverKey or PublicParameters

### ISSUE THREAD

**Issue by CPerezz** [dusk-network/plonk#332]
We currently have data structures in this crate whose serde times can be huge if they're verified correctly.

`PublicParameters` is an example of that. It can be really big and it always needs to be read from a file. At least each time we start the binary on which it is used.
Load `PublicParameters` which have a size of aprox `2^17 - 2^18` checking that each point that they contain relies indeed on the EC is expensive. And since the object is being read from a file and it's not coming from a polluted env (network for example) we should have a fn implemented that allows us to generate the structure from a raw in an "unchecked" way. 

Basically, just fetch from the file the internal representation of the structure in-memory.
That's at least my suggestion.

This means that `G1Affine` `G2Affine` and maybe another type in `dusk-bls12_381` will need to have a fn that allows us to store the internal representation of this data structures. So that we can se/deserialize them from this layout instead of from the bytes.

**Comment by vlopes11** [comment]
Fixed in https://github.com/dusk-network/plonk/pull/349
